EmployeeManagementService/EmployeeManagementService.Infrastructure/Persistence/EmployeeRepository.cs

```diff
@@ -9,12 +9,14 @@ namespace EmployeeManagementService.Infrastructure.Persistence
 {
     public interface IEmployeeRepository
     {
-        Task CreateEmployee(string firstName, string lastName, string username, string ssn, byte[] password, string role, bool? active = true);
+        Task CreateEmployee(string firstName, string lastName, string username, string ssn, byte[] password, string role, string a1, string a2, string city, string state, string zip, bool? active = true);
+        Task<Employee> GetAddressByEmployeeId(long id);

```

Comment:
No need for these. We can handle them all in GetEmployee, UpdateEmployee, and CreateEmployee